src/views/Home.jsx

```diff
@@ -2,27 +2,28 @@ import './Home.css';
 import { useNavigate } from 'react-router-dom';
 import { generateToken } from '@the-collab-lab/shopping-list-utils';
 
-export function Home({ setListToken }) {
+export function Home({ makeNewList }) {
 	const navigate = useNavigate();
```

Comment:
nice, I like useNavigate. What if we tried it in the app.jsx? Similar to the useEffect hook `if (!listToken) return;` you could try something like : 
```
if (listToken) 
{navigateTo('/list');
```

Definitely more of a nice to have, but wanted to suggest in case you're up for giving it a try!

Reply:
Thanks @chelseychubb ! I had lines 32-34 in App.js to try doing the initial redirect - is that working as expected? React did not like how `navigate` is not in the deps but that is necessary to prevent eternal redirects. What do you think?

Reply:
That makes total sense! Didn't catch that!